docusaurus/docs/reference/deployments/01-controller.md

```diff
@@ -6,24 +6,126 @@ sidebar_label: Controller
 
 import CliLogFormats from '../../_cli-log-levels-and-formats.md'
 
-This article provides some guidance for deploying a Ziti Controller. You can go back and read [the controller component introduction](/learn/introduction/03-components.md#openziti-controller) if needed.
+This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide. [Kubernetes is covered separately](/guides/kubernetes/hosting/kubernetes-controller.mdx).

```

Comment:
There's no need for the kubernetes covered separately comment. Let them discover via the sidebar